Source/Scene/Cesium3DTileset.js

```diff
@@ -1272,67 +1279,85 @@ define([
         }
     }
 
-    function updateTileDebugLabels(tileset, frameState) {
-        var selectedTiles = tileset._selectedTiles;
-        var length = selectedTiles.length;
-        tileset._tileDebugLabels.removeAll();
-        for (var i = 0; i < length; ++i) {
-            var tile = selectedTiles[i];
-            var boundingVolume = tile._boundingVolume.boundingVolume;
-            var halfAxes = boundingVolume.halfAxes;
-            var radius = boundingVolume.radius;
-
-            var position = Cartesian3.clone(boundingVolume.center, scratchCartesian);
-            if (defined(halfAxes)) {
-                position.x += 0.75 * (halfAxes[0] + halfAxes[3] + halfAxes[6]);
-                position.y += 0.75 * (halfAxes[1] + halfAxes[4] + halfAxes[7]);
-                position.z += 0.75 * (halfAxes[2] + halfAxes[5] + halfAxes[8]);
-            } else if (defined(radius)) {
-                var normal = Cartesian3.normalize(boundingVolume.center, scratchCartesian);
-                normal = Cartesian3.multiplyByScalar(normal, 0.75 * radius, scratchCartesian);
-                position = Cartesian3.add(normal, boundingVolume.center, scratchCartesian);
-            }
+    function addTileDebugLabel(tile, tileset) {
+        var boundingVolume = tile._boundingVolume.boundingVolume;
+        var halfAxes = boundingVolume.halfAxes;
+        var radius = boundingVolume.radius;
+
+        var position = Cartesian3.clone(boundingVolume.center, scratchCartesian);
```

Comment:
I should have left this comment earlier, but since there are two different ways of positioning labels now, maybe they can be broken out into separate functions.